index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <h1>Welcome To Microverse!</h1>
+  <p>
+    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Similique,
+    laudantium!
+  </p>
+
+  <h1>What is Pair Programming</h1>
```

Comment:
I think it would be better to use the h1 element just once time on the page.
With the h1 element, you can easily differentiate your code part, and its a good coding practice and also its improve your website's SEO